draft-ietf-rats-concise-ta-stores.md

```diff
@@ -74,9 +74,9 @@ Trust anchor (TA) stores may be used for several purposes in the Remote Attestat
 
 The RATS architecture {{RFC9334}} uses the definition of a trust anchor from {{RFC6024}}: "A trust anchor represents an authoritative entity via a public key and associated data.  The public key is used to verify digital signatures, and the associated data is used to constrain the types of information for which the trust anchor is authoritative." In the context of RATS, a trust anchor may be a public key or a symmetric key. This document focuses on trust anchors that are represented as public keys.
 
-The Concise Reference Integrity Manifest (CoRIM) {{I-D.draft-ietf-rats-corim}} specification defines a binary encoding for reference values using the Concise Binary Object Representation (CBOR) {{RFC8949}}. Amongst other information, a CoRIM may include key material for use in verifying evidence from an attesting environment (see section 3.11 in {{I-D.draft-ietf-rats-corim}}). The extension in this document aims to enable public key material to be decoupled from reference data for several reasons, described below.
+The Concise Reference Integrity Manifest (CoRIM) {{I-D.draft-ietf-rats-corim}} specification defines a binary encoding for reference values and endorsmenents using the Concise Binary Object Representation (CBOR) {{RFC8949}}. Amongst other information, a CoRIM may include key material for use in verifying evidence from an attesting environment (see {{Section 3.1.4.4 and 3.1.4.5 of I-D.draft-ietf-rats-corim}}). The extension in this document aims to enable public key material to be decoupled from reference data and other kinds of endorsement data for several reasons, described below.
```

Comment:
The usage of `reference data` here is ambigous and I suspect this was the problem ?

I suggest we re-word this to make it more clear!

Reply:
@hannestschofenig @muhammad-usama-sardar could you please share your thoughts?

Reply:
With the addition of endorsements, the first sentence is ok for me now. I still have trouble understanding the terms: 
- "reference data" in contrast with "reference values" 
- "endorsement data" in contrast with "endorsements"

Reply:
> With the addition of endorsements, the first sentence is ok for me now.

Cool, thanks for checking.

> I still have trouble understanding the terms:
> 
> * "reference data" in contrast with "reference values"
> * "endorsement data" in contrast with "endorsements"

They are synonyms.

Reply:
there is no word as `reference data` in the RATS architecture. We should not mint new words atleast in the specification!

Reply:
addressed in [74c1917](https://github.com/ietf-rats-wg/draft-wallace-rats-concise-ta-stores/pull/4/commits/74c1917405e76f8bb9b6b6bc749cd2a724ad174d)